### ISSUE TITLE
CSRF verification failed error

### ISSUE THREAD

**Issue by mtranchina** [chiefonboarding/ChiefOnboarding#268]
I am testing Chiefonboarding in a Docker Container on my Synology NAS and getting this CSRF verification failed error.  Does anyone have an idea on how to correct this error?
![image (28)](https://user-images.githubusercontent.com/20953028/213538410-6ae53c84-3ecd-4805-91e8-7a4a70ba3d74.png)

**Comment by GDay** [comment]
That's very strange. 

Could you try to open your site in incognito and see if you get it there too?

**Comment by mtranchina** [comment]
Same...

![image](https://user-images.githubusercontent.com/20953028/213576252-90684cda-add5-4bd3-9fad-35efeafeb744.png)

**Comment by GDay** [comment]
Ah, you are not using `https`. Have you set `HTTP_INSECURE` to `True` in your environment variables?

**Comment by mtranchina** [comment]
Checking...

**Comment by mtranchina** [comment]
Yup...That was the problem...When accessing from outside my network using DDNS and HTTPS it works fine...Using my internal IP HTTP gives me that error...

Thanks!

**Comment by mtranchina** [comment]
Now I just need to replace Slack with Mattermost! :-)

**Comment by Daandeve** [comment]
I have the exact same issue but the other way around, I can access it and login via the local ip, but via the remote domain I can access it but I can't login
![image](https://github.com/chiefonboarding/ChiefOnboarding/assets/35433138/6d973e19-0cb8-42ff-9234-e39dcdc0ac92)

**Comment by Kuemmelklaus** [comment]
Is there any way to prevent this from happening while accessing with an internal IP. I'm experiencing the same issue on a docker installation via a caddy reverse proxy:

![Image](https://github.com/user-attachments/assets/e7490b28-0d64-46c5-9029-116d4b178c88)
